test/integration/submission/migration_utils/stub-diffs.ts

```diff
@@ -400,40 +400,6 @@ export const migrationDiffs = [
       ],
     ],
   },
-  {
```

Comment:
what happened here ?

Reply:
There were two repeated `diff` section both are from version 1.0 to version 12.0, i deleted the duplicate.